pkg/ccl/streamingccl/streamingest/stream_ingestion_processor.go

```diff
@@ -135,13 +151,15 @@ func newStreamIngestionDataProcessor(
 		output:              output,
 		curBatch:            make([]storage.MVCCKeyValue, 0),
 		client:              streamClient,
+		input:               input,
 		bufferedCheckpoints: make(map[streamingccl.PartitionAddress]hlc.Timestamp),
 		timer:               timeutil.NewTimer(),
+		signalCh:            make(chan rowenc.EncDatum),
 	}
 
-	if err := sip.Init(sip, post, streamIngestionResultTypes, flowCtx, processorID, output, nil, /* memMonitor */
+	if err := sip.Init(sip, post, input.OutputTypes(), flowCtx, processorID, output, nil, /* memMonitor */
```

Comment:
Although in this case `input.OutputTypes() == streamIngestionResultsTypes` I think that this should remain as they were?